src/firefly/js/ui/tap/TableSelectViewPanel.jsx

```diff
@@ -184,7 +184,7 @@ export class TapSearchPanel extends PureComponent {
 
                         <div className='TapSearch__section'>
                             <div className='TapSearch__section--title'>1. TAP Service <HelpIcon helpId={tapHelpId('tapService')}/> </div>
-                            <div style={{flexGrow: 1, marginRight: 3, maxWidth: 1000}}>
+                            <div style={{flexGrow: 1, marginRight: 3, maxWidth: 1000, zIndex: 9999}}>
```

Comment:
Can you describe when this is needed?  I am looking at `dev` now, and it looks like it's working fine without the change.  With this change is when the Schema and Table input fields flow on top of the TAP Service dropdown.

Reply:
![image](https://user-images.githubusercontent.com/549395/105399943-21523a80-5bd9-11eb-8685-b7ec37da4b20.png)

See image ^. In the case when you are selecting the bottom of GAIA or the top of GAVO, the textarea is actually z-index=1 and it is above those divs (TapSearch__section) and selection won't actually work in that limited area, but it works in every other area of the dropdown.

Reply:
ah.  thanks.  i was looking at the wrong section.